web/src/components/localization-select/localization-select.css

```diff
@@ -0,0 +1,114 @@
+.localization-select {
+    position: relative;
+    font-family: 'Inter', sans-serif;
+
+    &::before {
+        content: '';
+        width: 100%;
+        height: 100%;
+        position: absolute;
+        background: url('./globe.svg') no-repeat left 13px center;
+        background-size: 17px 16px;
+        pointer-events: none;
+        z-index: var(--top-z-index);
+    }
+
+    &.with-down-arrow::after {
+        top: 0;
+    }
+
+    & select,
+    & .button {
+        border: 1.6px solid #e6e4e1;
+        border-radius: 2px;
+        padding-inline-start: 2.5rem;
+        background: white;
+        text-transform: uppercase;
+        font-size: var(--font-size-xs);
+        font-weight: 600;
+        letter-spacing: 0.8px;
+        color: var(--warm-grey);
+        cursor: pointer;
+    }
+
+    & .button {
```

Comment:
This way of doing styles is a bit dangerous because we can have classes leaking into other places. Something to think about in future.

Reply:
Can you expand a bit more on what risk you're envisioning?

Reply:
For example, here you might assume the element with the class `.button` in `LocalizationSelect` is only styled by this code here, but it's also styled by a file in `ui.css` which considering we don't link to that at any point in this component is confusing.